tests/common/alarm_test.go

```diff
@@ -32,17 +32,18 @@ func TestAlarm(t *testing.T) {
 	defer cancel()
 	clus := testRunner.NewCluster(ctx, t, config.ClusterConfig{ClusterSize: 1, QuotaBackendBytes: int64(13 * os.Getpagesize())})
 	defer clus.Close()
+	cc := clus.MustClient(config.ClientOption{})
 	testutils.ExecuteUntil(ctx, t, func() {
 		// test small put still works
 		smallbuf := strings.Repeat("a", 64)
-		if err := clus.Client().Put("1st_test", smallbuf, config.PutOptions{}); err != nil {
+		if err := cc.Put("1st_test", smallbuf, config.PutOptions{}); err != nil {
```

Comment:
Can you resolve this in a separate PR so as to make this one as simple & clean as possible?